Tests/scripts/validate_files_structure.py

```diff
@@ -464,6 +471,20 @@ def integration_valid_in_id_set(file_path, integration_set):
 
 
 def validate_committed_files(branch_name, is_circle):
+
+    # Get all files that have been modified & added to repo
+    secrets_file_paths = get_all_diff_text_files(branch_name, is_circle)
+    secrets_found = search_potential_secrets(secrets_file_paths)
+
+    if secrets_found:
+        print_error('Secrets were found in the following files:\n')
+        for file_name in secrets_found:
+            print_error('\nFile Name: ' + file_name)
+            print_error(json.dumps(secrets_found[file_name], indent=4))
+        print_error('Remove or whitelist secrets in order to proceed, then re-commit\n')
```

Comment:
need to address the case this was found only in circle build and not in the hooks (somehow..)
so let's add suggest gently to remove the file asap..